python/ql/test/library-tests/frameworks/FileSystemAccess/starlette.py

```diff
@@ -0,0 +1,5 @@
+from starlette.responses import FileResponse
+from baize.asgi import FileResponse as baizeFileResponse
```

Comment:
these two don't seem to have anything to do with each other, so could we split them into separate files please? :pray: